draft-ietf-quic-recovery.md

```diff
@@ -280,23 +294,28 @@ connection's final smoothed RTT value as the resumed connection's initial RTT.
 If no previous RTT is available, or if the network changes, the initial RTT
 SHOULD be set to 100ms.
 
-When a handshake packet is sent, the sender SHOULD set an alarm for the
-handshake timeout period.
+When CRYPTO_HS frames are sent, the sender SHOULD set an alarm for the
+handshake timeout period.  When the alarm fires, the sender MUST retransmit
+all unacknowledged CRYPTO_HS data by calling
+RetransmitAllUnackedHandshakeData(). On each consecutive firing of the
+handshake alarm without receiving an acknowledgement for a new packet,
+the sender SHOULD double the handshake timeout and set an alarm for this
+period.
 
-When the alarm fires, the sender MUST retransmit all unacknowledged handshake
-data, by calling RetransmitAllUnackedHandshakeData(). On each consecutive
-firing of the handshake alarm, the sender SHOULD double the handshake timeout
-and set an alarm for this period.
+When CRYPTO_HS frames are outstanding, the TLP and RTO timers are not active
+unless the CRYPTO_HS frames were sent at 1RTT encryption.
```

Comment:
I was intending it to be encryption level, since I thought it was necessary to send end of early data in a timely manner?  But maybe it doesn't matter that much? 

It's possible a signal that the handshake is complete would be preferable to monitoring what encryption level CRYPTO_HS is sent at, but I'll just make it consistent for now.

Reply:
Sounds reasonable.  I just wanted to check.  And yeah, we probably need to send EndOfEarlyData quickly.  It blocks access to 1-RTT data just like Finished.